drivers/overlay/encryption.go

```diff
@@ -43,17 +45,39 @@ func (k *key) String() string {
 	return ""
 }
 
+// hosts-pair spis
 type spi struct {
 	forward int
 	reverse int
 }
 
-func (s *spi) String() string {
+func (s spi) String() string {
 	return fmt.Sprintf("SPI(FWD: 0x%x, REV: 0x%x)", uint32(s.forward), uint32(s.reverse))
 }
 
+// hosts-pair keys
+type hpk struct {
+	forward []byte
+	reverse []byte
+}
+
+func (h hpk) String() string {
+	return fmt.Sprintf("Key(FWD: %s, REV: %s)",
```

Comment:
I don't think we should have code in our code base that prints the key. That is asking for trouble.